main.tf

```diff
@@ -293,6 +293,7 @@ module "karpenter" {
   node_iam_role_attach_cni_policy = false
   node_iam_role_additional_policies = {
     AmazonSSMManagedInstanceCore = "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore"
+    fluentbit                    = aws_iam_policy.fluentbit.arn
```

Comment:
Can you use IRSA please

Reply:
> Can you use IRSA please

will have a look. No idea how to do it.... but ok